lib/HALSim/src/Board.cpp

```diff
@@ -32,10 +32,16 @@
 /******************************************************************************
  * Includes
  *****************************************************************************/
+#if _WIN32
```

Comment:
Shouldn't it be "#ifdef ...." at all places?

Reply:
Yes, it behaves the same but #if <maybe undef> isn't MISRA compliant - changed.